nansat/mappers/mapper_opendap_mywave.py

```diff
@@ -0,0 +1,76 @@
+# Name:         mapper_opendap_mywave.py
+# Purpose:      Nansat mapping for MyWaveWAV data provided by MET.NO
+# Author:       Artem Moiseev
+# Licence:      This file is part of NANSAT. You can redistribute it or modify
+#               under the terms of GNU General Public License, v.3
+#               http://www.gnu.org/licenses/gpl-3.0.html
+
+from nansat.mappers.opendap import Opendap
+from nansat.exceptions import WrongMapperError
+from nansat.nsr import NSR
+from netCDF4 import Dataset
+from datetime import datetime
+import numpy as np
+import json
+import pythesint as pti
+import os
+
+
+class Mapper(Opendap):
+
+    baseURLs = ['http://thredds.met.no/thredds/dodsC/fou-hi/mywavewam4archive']
+    timeVarName = 'time'
+    xName = 'rlon'
+    yName = 'rlat'
+    timeCalendarStart = '1970-01-01'
+
+    def __init__(self, filename, gdal_dataset, gdal_metadata, date=None,
+                 ds=None, bands=None, cachedir=None, *args, **kwargs):
+
+        self.test_mapper(filename)
+        timestamp = date if date else self.get_date(filename)
+        ds = Dataset(filename)
+        try:
+            self.srcDSProjection = NSR(ds.variables['projection_3'].proj4 +
+                                       ' +to_meter=0.0174532925199 +wktext')
+        except KeyError:
+            raise WrongMapperError
+
+        self.create_vrt(filename, gdal_dataset, gdal_metadata, timestamp, ds, bands, cachedir)
+        mm = pti.get_gcmd_instrument('Computer')
+        ee = pti.get_gcmd_platform('MODELS')
+        self.dataset.SetMetadataItem('instrument', json.dumps(mm))
+        self.dataset.SetMetadataItem('platform', json.dumps(ee))
+        self.dataset.SetMetadataItem('Data Center', 'NO/MET')
+        self.dataset.SetMetadataItem('Entry Title', str(ds.getncattr('title')))
+
```

Comment:
Some missing metadata should be added:

* `time_coverage_*`
* `ISO_topic_category`
* `set_metadata('gcmd_location', json.dumps(pti.get_gcmd_location('sea surface')))`

Reply:
`time_coverage_*` metadata are added in opendap.py file during an execution of the `create_vrt` method